src/Providers/DataGrid/Wijmo/Grid/FlexGrid.ts

```diff
@@ -299,6 +311,10 @@ namespace Providers.DataGrid.Wijmo.Grid {
 
             this._fBuilder.dispose();
 
+            this._provider.resizedColumn.removeHandler(
+                this._updateColumnWidth.bind(this)
+            );
```

Comment:
This will not remove the handler, as the `.bind(this)` will generate a new "id" for the function each time